cloudinstall/multi_install.py

```diff
@@ -196,6 +199,39 @@ def drop_privileges(self):
         os.setregid(pwnam.pw_gid, pwnam.pw_gid)
         os.setreuid(pwnam.pw_uid, pwnam.pw_uid)
 
+    def add_bootstrap_to_no_proxy(self):
+        """Finds bootstrap node IP and adds it to the current setting of
+        no-proxy in the juju env.
+        """
+        out = utils.get_command_output(
+            "{0} juju status".format(
```

Comment:
as discussed on irc, mostly because we don't currently have an api client in the install section (openstack-install) of the installer (as opposed to openstack-status). This approach was simpler to get going.